package.json

```diff
@@ -39,9 +39,8 @@
     "uuid": "^8.3.2"
   },
   "dependencies": {
-    "@asl/constants": "^1.2.2",
-    "@asl/schema": "^10.16.0",
-    "@asl/service": "^9.2.0",
```

Comment:
Is constants just not needed?

Reply:
Apparently not! It doesn't get referenced in the project anywhere and tests pass, so I'm deciding to say it's not needed. My guess is it was added with the idea it would eventually be used, but it never was.